### PR TITLE
Feat/add patpass comercio

### PR DESCRIPTION
**Pull request by themakunga**
add the views and controllers of `create inscription` and `status`

### CODE REVIEW

**Comment by goncafa** [comment]
@TheYakuza You should first resolve the branch conflicts before we could review this code

**Comment by themakunga** [comment]
@goncafa Resolved!

**Review by goncafa — APPROVED** [review]
Looks good to me 🎉

File: transbank-sdk-dotnet-rest-example/Controllers/PatpassComercioController.cs [hunk 1]
```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Mvc;
+using Transbank.Common;
+using Transbank.Patpass.PatpassComercio;
+
+namespace transbanksdkdotnetrestexample.Controllers
```
Comment (leosoto) [inline transbank-sdk-dotnet-rest-example/Controllers/PatpassComercioController.cs:8]: Is this a common way to name namespaces in C# (I hope it isn't!)

File: transbank-sdk-dotnet-rest-example/Views/Home/Index.cshtml [hunk 2]
```diff
@@ -104,4 +104,18 @@
...
+        <td>
+            <h3>Patpass Comercio</h3>
+        </td>
+        <td>
+            <h3>@Html.ActionLink("Iniciar Inscripcion", "Start", "PatpassComercio")</h3>
+        </td>
+        <td>
+            <h3>-</h3>
+        </td>
+        @* <td> *@
```
Comment (leosoto) [inline transbank-sdk-dotnet-rest-example/Views/Home/Index.cshtml:117]: If that's commented-out code then it shouldn't be committed at all

**Pull request merged**